### DISCUSSION TITLE
Is there any way to do relative import for `.drift` files while passing it as

### DISCUSSION THREAD

**Discussion by Vedsaga** [simolus3/drift#2816] [Q&A] [answered]
# Actual

```dart

@DriftDatabase(
  include: {
    'package:daily_routine_tracker/config/app_db/schema/profile_table.drift',
  },
)
```

to include `.drift` file in AppDb, currently specifying full path. 

Project folder structure is,
```text
-- lib
----  config
------ app_db
-------- schema
---------- `profile_table.drift`
```

now, inside the `config folder` we have `config.dart`
```
-- lib
----  config
------- `config.dart` 
```

in that we are exporting all the files related to the app_db like below
`config.dart file`
```dart
export 'app_db/app_db.dart';
export 'app_db/db_typedef.dart';
export 'app_db/query/profile_table_query.dart';
```

# Expecting
So, is it possible to make below import work?

```dart

import 'package:daily_routine_tracker/config/config.dart';


@DriftDatabase(
  include: {
    'profile_table.drift',
  },
)
```

**Comment by simolus3** [comment, marked as answer]
Is the database file in the same folder as the drift file? The relative import should work in that case, you can also do `'schema/profile_table.drift'` if it's in a subdirectory. Generally, imports and includes in drift are designed to behave like Dart imports.

**Reply by Vedsaga** [reply]
`schema/profile_table.drift` works, As db-file & schema folder where in same dir which is `config`
